controllers/secretgenerator/secret.go

```diff
@@ -8,19 +8,20 @@ import (
 	"strconv"
 )
 
+//nolint:golint,revive,stylecheck //CAPS is preferred for const
 const (
-	SECRET_NAME_ANNOTATION         = "secret-generator.opendatahub.io/name"               //nolint
-	SECRET_TYPE_ANNOTATION         = "secret-generator.opendatahub.io/type"               //nolint
-	SECRET_LENGTH_ANNOTATION       = "secret-generator.opendatahub.io/complexity"         //nolint
-	SECRET_OAUTH_CLIENT_ANNOTATION = "secret-generator.opendatahub.io/oauth-client-route" //nolint
-	SECRET_DEFAULT_COMPLEXITY      = 16                                                   //nolint
+	SECRET_NAME_ANNOTATION         = "secret-generator.opendatahub.io/name"
+	SECRET_TYPE_ANNOTATION         = "secret-generator.opendatahub.io/type"
+	SECRET_LENGTH_ANNOTATION       = "secret-generator.opendatahub.io/complexity"
+	SECRET_OAUTH_CLIENT_ANNOTATION = "secret-generator.opendatahub.io/oauth-client-route"
+	SECRET_DEFAULT_COMPLEXITY      = 16
 
 	letterRunes = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
 
-	errEmptyAnnotation        = "secret annotations is empty"
-	errNameAnnotationNotFound = "name annotation not found in secret"
-	errTypeAnnotationNotFound = "type annotation not found in secret"
-	errUnsupportedType        = "secret type is not supported"
+	ErrEmptyAnnotation        = "secret annotations is empty"
+	ErrNameAnnotationNotFound = "name annotation not found in secret"
+	ErrTypeAnnotationNotFound = "type annotation not found in secret"
+	ErrUnsupportedType        = "secret type is not supported"
```

Comment:
any reason to expose these?  or just to be used in the test?

Reply:
so lint was expecting us to have `_test` at the end of each test packages.. and the tests were using these variables. So I have to expose them

Reply:
i am fine with moving all test cases into a dedicated test package.
but in order to make test passing then expose unnecessary variable is not the right approach.
i think this is not in the scope of this PR, since it had been done in the way at first.

Reply:
ok so for test cases, shall I duplicate these vars in the test package??